### DISCUSSION TITLE
Be able to draw in editor

### DISCUSSION THREAD

**Discussion by saumon-brule** [ppy/osu#30737] [Ideas]
Hi.
As I started to map some songs on osu (lazer) I often think about patterns and I often have ideas for later like "where is going the map", "How will look the map when I will put all the circles" but I can't remember all and I think that it would be very convenient to have a way to draw some "draft" (like with the smoke when playing) to remember these ideas and not just put some random circles and sliders in the editor.
I think that permanent drawing that would not be displayed when the map is played from somewhere else than the editor may be the best way to implement this.

**Comment by vatei** [comment]
That sounds super useful yeah, a way to type notes would also be nice, not just drawings, maybe with the username attached (good for collabs)
